packages/menu/src/Menu.ts

```diff
@@ -443,6 +496,62 @@ export class Menu extends SizedMixin(SpectrumElement, { noDefaultSize: true }) {
         }
     }
 
+    /**
+     * Handles touchstart events for iPad scroll detection.
+     *
+     * Records the initial touch position and timestamp to establish a baseline
+     * for detecting scroll gestures. Only processes single-touch events to
+     * avoid interference with multi-touch gestures.
+     *
+     * @param event - The TouchEvent from the touchstart event
+     */
+    private handleTouchStart(event: TouchEvent): void {
+        if (event.touches.length === 1) {
+            this.touchStartY = event.touches[0].clientY;
+            this.touchStartTime = Date.now();
+            this.isScrolling = false;
+        }
+    }
+
+    /**
+     * Handles touchmove events for iPad scroll detection.
+     *
+     * Calculates the vertical movement distance and time elapsed since touchstart.
+     * If the movement exceeds the threshold (10px) and happens within the time
+     * threshold (300ms), it marks the interaction as scrolling. This helps
+     * distinguish between intentional scroll gestures and accidental touches.
+     *
+     * @param event - The TouchEvent from the touchmove event
+     */
+    private handleTouchMove(event: TouchEvent): void {
+        if (event.touches.length === 1) {
+            const currentY = event.touches[0].clientY;
+            const deltaY = Math.abs(currentY - this.touchStartY);
+            const deltaTime = Date.now() - this.touchStartTime;
+
+            if (
+                deltaY > this.scrollThreshold &&
+                deltaTime < this.scrollTimeThreshold
+            ) {
+                this.isScrolling = true;
+            }
+        }
+    }
+
+    /**
+     * Handles touchend events for iPad scroll detection.
+     *
+     * Resets the scrolling state after a short delay (100ms) to allow for
+     * any final touch events to be processed. This delay prevents immediate
+     * state changes that could interfere with the selection logic.
+     */
+    private handleTouchEnd(): void {
+        // Reset scrolling state after a short delay
+        setTimeout(() => {
```

Comment:
This is where we can set the start variables to undefined instead of manually updating the `isScrolling` boolean. It will also ensure our start values are reset appropriately when we're done so that they're not returning old values when used.